src/MaterialEditor.Core/Core.MaterialEditor.Hooks.cs

```diff
@@ -50,10 +50,17 @@ private static void GetBodyRendererList(GameObject gameObject, List<Renderer> re
         }
 
         private static readonly Dictionary<GameObject, List<Renderer>> _RendererLookup = new Dictionary<GameObject, List<Renderer>>();
+
+        internal static void ClearCache(GameObject gameObject = null)
+        {
+            if (gameObject == null) _RendererLookup.Clear();
+            else if (_RendererLookup.ContainsKey(gameObject)) _RendererLookup.Remove(gameObject);
```

Comment:
```suggestion
            else _RendererLookup.Remove(gameObject);
```
No need to check first, just remove.